docs/SalesforceMultiObjects-batchsource.md

```diff
@@ -29,6 +29,10 @@ A Salesforce connected application must be created in order to get a client secr
 
 **Login Url:** Salesforce OAuth2 login url.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use.
```

Comment:
Same issue here. The SObject Name examples are now under Connection property.

Reply:
Changed the sequence.